go.mod

```diff
@@ -4,6 +4,7 @@ go 1.21
 
 require (
 	github.com/confluentinc/confluent-kafka-go/v2 v2.3.0
+	github.com/ethereum/go-ethereum v1.10.26
```

Comment:
This repo needs to remain chain-agnostic. What is geth being used for?

Reply:
I moved `utils.big` which is using `goeth/hexutils`. Do you think this is an issue?

Reply:
We'll have to swap it out, but hex is easy - most of that package is just wrapping std lib, and we can copy whatever we need to.